bool graficar_realtime
int64 step
int64 idcall
